src/views/Activate/ActivatePassport.js

```diff
@@ -68,7 +67,7 @@ export default function ActivatePassport() {
     <LocalRouterProvider value={router}>
       <View full={fullView}>
         <Grid gap={gap} className="mt8 mb10">
-          <Grid.Item half={leftHalf} full={full}>
+          <Grid.Item half={half && 1} full={full}>
```

Comment:
This seems like a downgrade, readability-wise, from the earlier version.

Reply:
I actually like it, since I can look at the components here and know that this specific Grid.Item will be the left side and the other will be the right, without having to go check what the values of `leftHalf` and `rightHalf` will be.